analysis/yearling_rulesets.R

```diff
@@ -0,0 +1,135 @@
+library(googleCloudStorageR)
+library(tidyverse)
+library(lubridate)
+library(waterYearType)
+library(viridis)
+library(stringr)
+
+gcs_auth(json_file = Sys.getenv("GCS_AUTH_FILE"))
+gcs_global_bucket(bucket = Sys.getenv("GCS_DEFAULT_BUCKET"))
+
+# Get data for google bucket
+gcs_get_object(object_name = "jpe-model-data/daily_catch_unmarked.csv",
+               bucket = gcs_get_global_bucket(),
+               saveToDisk = "analysis/data/daily_catch_unmarked.csv",
```

Comment:
When I try to run this it throws an error because the analysis/data folder doesn't exist. Is this something specific to my computer, or should this line be be data/model-data ?

Reply:
it should work after you add a data folder within analysis. i should have pushed the folder up but since we ignore csv there was nothing in the folder to push and i missed it. sorry!